cmake/std/atdm/cts1empire/environment_new.sh

```diff
@@ -55,8 +62,8 @@ fi
 export OMPI_CXX=`which icpc`
 export OMPI_CC=`which icc`
 export OMPI_FC=`which ifort`
-export ATDM_CONFIG_LAPACK_LIBS="-mkl"
-export ATDM_CONFIG_BLAS_LIBS="-mkl"
+export ATDM_CONFIG_LAPACK_LIBS="-mkl -L${MKL_LIB}"
```

Comment:
Where is `MKL_LIB` coming from?

Reply:
I think `-mkl` is a special compiler/linker flag that is directly understood by the Intel compiler and it does not need to know a library path with `-L` to find the MKL libraries associated with the compiler. Perhaps `-mkl` just instructs the Intel compiler to put the right `-l<lib>` arguments on the link line but you can change what MKL implementation is used by setting `-L<mkl-lib-dir>`?  Is that what the SPARC modules are doing here on CTS-1?

Reply:
That was a further hack for a test build I've done with Intel 2021.3.0 and the corresponding MKL.

The `-mkl` flag (deprecated, per Intel, in favor of `-qmkl`) is supposed to add the appropriate includes, library paths, and linked libraries to the command lines it generates, but it seems not to in the one environment I have available to test it on, SNL's Eclipse cluster. I got linking errors for it not being able to find the MKL libraries at link time.

At any rate, that's irrelevant to this PR, which I've reduced back to just the Intel 19 changes